config/locales/simple_form.en.yml

```diff
@@ -61,4 +61,5 @@ en:
         repo_name: E.g. rails/rails, jollygoodcode/twemoji
         travis_token: Retrieve Travis Token from the Profile tab on your Accounts page
         codeship_uuid: Retrieve Project UUID from https://codeship.com/projects/[PROJECTID]/configure
+        circleci_token: "Add an CircleCI Token your account dashboard. https://circleci.com/docs/api#authentication"
```

Comment:
I think we should also add to the hint >> `(required for private repos only)`. wdyt?